pkg/aspect/accessLogsManager.go

```diff
@@ -32,98 +33,92 @@ type (
 	accessLogsManager struct{}
 
 	accessLogsWrapper struct {
-		logName   string
-		inputs    map[string]string // map from param to expr
-		aspect    adapter.AccessLogsAspect
-		attrNames []string
-		template  *template.Template
+		name          string
+		aspect        adapter.AccessLogsAspect
+		labels        map[string]string // label name -> expression
+		template      *template.Template
+		templateExprs map[string]string // template variable -> expression
 	}
 )
 
 const (
 	// TODO: revisit when well-known attributes are defined.
 	commonLogFormat = `{{or (.originIp) "-"}} - {{or (.source_user) "-"}} ` +
-		`[{{or (.timestamp.Format "02/Jan/2006:15:04:05 -0700") "-"}}] "{{or (.apiMethod) "-"}} ` +
+		`[{{or (.timestamp.Format "02/Jan/2006:15:04:05 -0700") "-"}}] "{{or (.method) "-"}} ` +
 		`{{or (.url) "-"}} {{or (.protocol) "-"}}" {{or (.responseCode) "-"}} {{or (.responseSize) "-"}}`
 	// TODO: revisit when well-known attributes are defined.
 	combinedLogFormat = commonLogFormat + ` "{{or (.referer) "-"}}" "{{or (.user_agent) "-"}}"`
 )
 
-var (
-	// TODO: revisit when well-known attributes are defined
-	commonLogAttributes = []string{"originIp", "source_user", "timestamp", "apiMethod", "url", "protocol", "responseCode", "responseSize"}
-	// TODO: revisit when well-known attributes are defined
-	combinedLogAttributes = append(commonLogAttributes, "referer", "user_agent")
-)
+var bufferPool = sync.Pool{
```

Comment:
Yes we should. My follow on PR for updating application logs uses the same thing (in fact, when I merge them I'll have to delete one of the two pools since we're in the same package).